src/hotspot/share/classfile/verifier.cpp

```diff
@@ -31,7 +31,9 @@
 #include "classfile/stackMapTableFormat.hpp"
 #include "classfile/symbolTable.hpp"
 #include "classfile/systemDictionary.hpp"
+#if INCLUDE_CDS
```

Comment:
Conditional includes go after the unconditional includes.  See HotSpot Style Guide.

Reply:
Done. Thanks for the reminder.